ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/AdminResponseHandlers.java

```diff
@@ -335,8 +335,8 @@ static boolean isConnectErrorResponse(final JsonObject ksqlEntity) {
           source.getString("valueFormat"),
           formatQueries(source.getJsonArray("readQueries")),
           formatQueries(source.getJsonArray("writeQueries")),
-          Optional.ofNullable(source.getString("timestamp")),
-          Optional.ofNullable(source.getString("windowType")),
+          Optional.ofNullable(emptyToNull(source.getString("timestamp"))),
+          Optional.ofNullable(emptyToNull(source.getString("windowType"))),
```

Comment:
is this pertinent to this PR?

Reply:
No but someone introduced a small bug in the meanwhile.

Reply:
Seems to overlap with https://github.com/confluentinc/ksql/pull/7475

Reply:
Ah, yeah.  I was just trying to fix everything in one fell swoop.